common/app/common/pages/BagItemData.java

```diff
@@ -0,0 +1,31 @@
+package common.pages;
+
+import java.util.List;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toList;
+
+public class BagItemData {
```

Comment:
I would appreciate a little Javadoc comment what class is about.

Is the bag the same as the cart?